### ISSUE TITLE
What do the EMS QA_INDEX_CODE values mean?

### ISSUE THREAD

**Issue by joethorley** [bcgov/water-quality-analysis#4]
```
> unique(ems$QA_INDEX_CODE)
[1] "C" "D" "F" "B"
```

**Comment by stephhazlitt** [comment]
Good question. I will find out.

**Comment by joethorley** [comment]
QA Index of B
1.  If a result letter of > or < is reported then assign an index of B
2.  If confidence interval and method detection limit not provided then assign an index of B
3.  If result values are determined to be rounded, maximum index of B
4.  For all taxonomic results set maximum index of B

QA Index of C
1.  If Taxonomic Keys not recorded for sample, maximum index of C
2.  If a Lab Batch ID is not provided for result, maximum index of C

QA Index of D
1.  If a contract lab does not provide preservation and media codes, maximum index of D

QA Index of E
1.  If no analytical date for result where a max hold time has been defined, maximum index of E

QA Index of F
1.  Failed Confidence Interval Conversion (Text To Numeric), maximum index of F
2.  Failed Result Conversion (Text to Numeric), maximum index of F
3.  Failed pH check. Value out of range 0 to 14, maximum index of F
4.  Failed Result Unit Conversion to Standard Units, maximum index of F
5.  If the sample does not have an EMS ID assigned, maximum index of F
6.  Other validation rules where dissolved > total
e.g. Validation Failed:  CU-D > CU-T, maximum index of F
